### DISCUSSION TITLE
Is Atkin respiration appropriate for C4 plants?

### DISCUSSION THREAD

**Discussion by wwieder** [ESCOMP/CTSM#1935] [Ideas]
CLM5 already uses autotrophic respiration from Atkin 2017, but in [updating FATES to use this scheme ](https://github.com/NGEET/fates/pull/931)@ckoven raised the question if the Atkin scheme is appropriate for C4 grasses?  This is a parallel discussion we can have on the CLM side too.  

@dlawrenncar noted this may be worth considering on the CTSM side too, and potentially discussing at a CLM science call.
@danicalombardozzi or Gordon may have ideas here?

**Comment by ckoven** [comment]
Thanks @wwieder. One minor correction: CLM5 uses the Atkin acclimation model, but not the complete Atkin et al. 2017 scheme.  The distinction is that there are two parts to the more recent parameterization: a basal reference-temperature respiration rate lmr25top that acclimates based on growing temperatures, and a short-term temperature sensitivity (based on Heskel et al 2016).  The first term is what CLM5 uses, but multiplied with the older short-term sensitivity from prior versions of CLM; the Atkin 2017 parameterization includes both terms.

**Reply by wwieder** [reply]
Thanks for clarifying!
